### PR TITLE
Adding VSO build scripts

### PR DESCRIPTION
**Pull request by jasonsandlin**

### CODE REVIEW

File: Build/BuildPackage.ps1 [hunk 1]
```diff
@@ -25,9 +25,11 @@ $logFile = Join-Path $PSScriptRoot BuildPackage.log
 
 Write-Host "Exporting Xbox Live Unity Plugin to " -NoNewline
 Write-Host $packagePath -ForegroundColor Green
+Write-Host "$($unity) -batchmode -logFile '$($logFile)' -projectPath '$($projectPath)' -exportPackage '$($exportAssetPath)' '$($packagePath)' -quit"
```
Comment (veleek) [inline Build/BuildPackage.ps1]: @jasonsandlin You don't need the extra $() around those variables.  You only need that when you need to evaluate something like $($unity.Length)

File: Utilities/VSOBuildScripts/postBuildScript.cmd [hunk 2]
```diff
@@ -0,0 +1,43 @@
+if "%1" == "local" goto testlocal
```
Comment (veleek) [inline Utilities/VSOBuildScripts/postBuildScript.cmd]: These build scripts should probably go into the Build folder along with the rest of the stuff.  Can the VSO build run powershell scripts?  We should probably keep the build scripts a consistent type.

Comment (jasonsandlin) [inline Utilities/VSOBuildScripts/postBuildScript.cmd]: This is consistent with https://github.com/Microsoft/xbox-live-api and the rest of the XBL github repos.

File: Build/BuildPackage.ps1 (hunk 1 above)
Comment (jasonsandlin) [inline Build/BuildPackage.ps1]: Ah, thanks

**Pull request merged**